src/reducers/__tests__/serviceRequest.test.js

```diff
@@ -0,0 +1,17 @@
+import reducer from '../serviceRequests';
+import { RECEIVE_SERVICEREQUESTS } from '../../actions/index';
+
+describe('serviceRequest reducer', () => {
+  it('should return initial state', () => {
```

Comment:
Update these initial state test to check for 
- [ ] call reducer with no args `reducer()`
- [ ] call reducer `reducer(undefined)`
- [ ] call reducer `reducer(undefined,{})`
- [ ] call reducer `reducer(undefined,{type:invalid})`